docs/how-to-guides/new-user-guides/backup-restore-and-disaster-recovery/migrate-rancher-to-new-cluster.md

```diff
@@ -140,6 +140,26 @@ Install the [`rancher-backup chart`](https://github.com/rancher/backup-restore-o
 
    1. Once the Restore resource has the status `Completed`, you can continue the cert-manager and Rancher installation.
 
+:::note Important:
+
+When migrating Rancher between different Kubernetes distributions (e.g. from K3s to RKE2), the object representing the local cluster has to be modified to allow Rancher to detect the new distribution. After the restoration is completed, and **before** bringing up rancher on the new cluster, edit the local cluster object:
```

Comment:
Can we clarify that the workaround is specifically for cases where the local cluster transitions between RKE1, RKE2, and K3s? This will help minimize the risk of customers attempting it on other distributions and encountering issues and then reaching out to the support for help.

Reply:
The workaround should also be done in other cases. If you migrate e.g. between another distribution and k3s, you'd still get the wrong driver (and some other fields) and wouldn't get the k3s upgrade configuration along with it.

Reply:
I see. So the workaround is universally necessary regardless of the original and the new distribution of the local cluster.

Reply:
I added a "any of" to emphasize this.